tests/unit/test_sonata_config.py

```diff
@@ -1,146 +1,106 @@
-    assert SimConfig.run_conf['Celsius'] == 35
-    assert SimConfig.run_conf['V_Init'] == -75
-    assert SimConfig.run_conf['SpikeLocation'] == 'soma'
+    run_conf = SimConfig.run_conf
+    assert run_conf['CircuitTarget'] == 'Mosaic'
+    assert run_conf['Simulator'] == 'NEURON'
+    assert run_conf['Duration'] == 50.0
+    assert run_conf['Dt'] == 0.1
+    assert run_conf['Celsius'] == 35
+    assert run_conf['V_Init'] == -65
+    assert run_conf['SpikeLocation'] == 'soma'
+    assert run_conf['ExtracellularCalcium'] == 1.2
 
-    # output section
-    assert SimConfig.run_conf['SpikesFile'] == 'spikes.h5'
-    assert SimConfig.run_conf['SpikesSortOrder'] == 'by_time'
 
-    # reports section
-    soma_report = SimConfig.reports['soma_report']
-    assert soma_report['Target'] == 'l4pc'
-    assert soma_report['Type'] == 'compartment'
-    assert soma_report['ReportOn'] == 'v'
-    assert soma_report['Compartments'] == 'center'
-    assert soma_report['Sections'] == 'soma'
-    assert soma_report['Scaling'] == 'Area'
-    assert soma_report['StartTime'] == 0.0
-    assert soma_report['EndTime'] == 50.0
-    assert soma_report['Dt'] == 0.1
-    assert soma_report['Enabled']
-    compartment_report = SimConfig.reports['compartment_report']
-    assert soma_report['FileName'] == str(Path(SimConfig.run_conf["OutputRoot"]) / "soma_report.h5")
-    assert compartment_report['Target'] == 'l4pc'
-    assert compartment_report['Type'] == 'compartment'
-    assert compartment_report['ReportOn'] == 'v'
-    assert compartment_report['Compartments'] == 'all'
-    assert compartment_report['Sections'] == 'all'
-    assert compartment_report['Scaling'] == 'Area'
-    assert compartment_report['StartTime'] == 0.0
-    assert compartment_report['EndTime'] == 10.0
-    assert compartment_report['Dt'] == 0.1
-    assert compartment_report['Enabled']
-    assert compartment_report['FileName'] == str(
-        Path(SimConfig.run_conf["OutputRoot"]) / "my_compartment_report.h5"
-        )
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "conditions": {
+                "extracellular_calcium": 1.2,
+                "mechanisms": {
+                    "GluSynapse": {
+                        "cao_CR": 2.0,
+                    }
+                }
+            }
+        }
+    },
+], indirect=True)
+def test_extracellular_calcium_warning(create_tmp_simulation_config_file, caplog):
+    from neurodamus.core.configuration import SimConfig
+    from neurodamus.utils.logging import setup_logging
 
-    # conditions section
-    conditions = list(SimConfig._simulation_config.Conditions.values())[0]
-    assert conditions['init_depleted_ProbAMPANMDA_EMS'] is False
-    assert conditions['minis_single_vesicle_ProbAMPANMDA_EMS'] is True
-    assert conditions['randomize_Gaba_risetime'] == 'False'
+    setup_logging.logging_initted = True
```

Comment:
I am curious, what would happen without this setting ?

Reply:
I guess `capsys` gets the warning msg anyway. In some tests we check that instead. 
But I am fine with this`caplog`. I will close this conversation.